src/coreclr/src/System.Private.CoreLib/src/System/Reflection/RuntimeModule.cs

```diff
@@ -112,13 +104,26 @@ public override byte[] ResolveSignature(int metadataToken)
                     {
                         ConstArray sig = MetadataImport.GetMemberRefProps(tk);
 
-                        if (*(MdSigCallingConvention*)sig.Signature.ToPointer() == MdSigCallingConvention.Field)
+                        if (*(MdSigCallingConvention*)sig.Signature == MdSigCallingConvention.Field)
                             throw new ArgumentException(SR.Format(SR.Argument_ResolveMethod, tk, this),
                                 nameof(metadataToken));
                     }
                 }
 
-                IRuntimeMethodInfo methodHandle = ModuleHandle.ResolveMethodHandleInternal(GetNativeHandle(), tk, typeArgs, methodArgs);
+                RuntimeTypeHandle[]? typeArgs = null;
+                RuntimeTypeHandle[]? methodArgs = null;
+                if (genericTypeArguments is not null && genericTypeArguments.Length > 0)
```

Comment:
Nit: The pattern `x is not null && x.Length > 0` could be simply `x?.Length > 0`. We do that [here](https://github.com/dotnet/runtime/blob/191ea066b157103d6332b2cf3e986b055fda3bcd/src/libraries/System.Private.CoreLib/src/System/Text/UTF8Encoding.Sealed.cs#L77), for instance.

Reply:
Changed